webapi/Extensions/SemanticKernelExtensions.cs

```diff
@@ -47,14 +59,19 @@ public static WebApplicationBuilder AddSemanticKernelServices(this WebApplicatio
                 var kernel = provider.GetCompletionKernel();
 
                 sp.GetRequiredService<RegisterSkillsWithKernel>()(sp, kernel);
+                sp.GetService<KernelSetupHook>()?.Invoke(sp, kernel);
```

Comment:
After some quick parsing, I get it.   A comment might not hurt.

Reply:
Added a comment